GenAiProject/AiChatApp/templates/AiChatApp/register.html

```diff
@@ -1,50 +1,79 @@
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            width: 100%;
+            max-width: 450px; /* Aumentar a largura */
+            z-index: 1;
+            background-color: white;
+            padding: 20px; /* Diminuir a altura (menor padding) */
+            border-radius: 10px;
+            box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
+        }
+        .left-side img {
+            max-width: 400px;
+        }
+        .right-side img {
+            max-width: 600px;
+        }
+        .error-message {
+            color: red;
+        }
+    </style>
 </head>
 <body>
-    <h2>Criar Conta</h2>
-    <p>Preencha os campos abaixo para criar uma nova conta.</p>
+    <div class="split-container">
+        <div class="left-side">
+            <img src="{% static 'AiChatApp/img/Coding.png' %}" alt="Imagem Coding">
+        </div>
+        <div class="right-side">
+            <img src="{% static 'AiChatApp/img/Dynamics.png' %}" alt="Imagem Dynamics">
+        </div>
+    </div>
 
-    <form method="post">
-        {% csrf_token %}
-    
-        <!-- Adicionar bloco para mensagens de erro -->
-        {% if form.errors %}
-            <div style="color: red;">
-                {% for field in form %}
-                    {% for error in field.errors %}
-                        <p>{{ error }}</p>
-                    {% endfor %}
-                {% endfor %}
-                {% for error in form.non_field_errors %}
-                    <p>{{ error }}</p>
-                {% endfor %}
-            </div>
-        {% endif %}
-    
-        <label for="id_username">Nome de Usuário:</label>
-        <input type="text" id="id_username" name="username" placeholder="Escolha um nome de usuário" required>
-        <small>Utilize apenas letras, números e os caracteres @/./+/-/_.</small><br><br>
-    
-        <label for="id_email">Email:</label>
-        <input type="email" id="id_email" name="email" placeholder="seuemail@exemplo.com" required>
-        <small>Insira um endereço de email válido.</small><br><br>
-    
-        <label for="id_password1">Senha:</label>
-        <input type="password" id="id_password1" name="password1" placeholder="Crie uma senha" required>
-        <small>A senha deve ter pelo menos 8 caracteres.</small><br><br>
-    
-        <label for="id_password2">Confirmar Senha:</label>
-        <input type="password" id="id_password2" name="password2" placeholder="Repita a senha" required>
-        <small>Digite a mesma senha para verificação.</small><br><br>
-    
-        <button type="submit">Cadastrar</button>
-    </form>
-    
-    <p>Já possui uma conta? <a href="{% url 'login' %}">Faça login aqui</a>.</p>
+    <div class="form-container">
+        <h2>Cadastro</h2>
+        <p>Preencha os campos abaixo para criar uma nova conta.</p>
+
+        <form method="post">
+            {% csrf_token %}
+            {{ form.as_p }}
+
```

Comment:
Nessa tela, acredito que faltam os erros relacionados ao register https://stackoverflow.com/questions/56187339/how-to-display-form-errors-message-properly-in-django

E achei que ficou informação demais, poderia ser um pouco mais clean esse form